ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -642,3 +642,43 @@ def verify_odf_without_ocs_in_installed_operator(self) -> tuple:
             expected_text="OpenShift Container Storage",
         )
         return odf_operator_presence, ocs_operator_presence
+
+    def verify_storage_clients_page(self):
+        """
+        Verify storage clients page in UI
+
+        """
+        self.refresh_web_console()
+        self.navigate_to_storageclients_page()
+        strings_storage_clients_tab = ["Storage clients"]
+        self.verify_page_contain_strings(
+            strings_on_page=strings_storage_clients_tab, page_name="storage clients"
+        )
+        self.do_click(
+            self.validation_loc["generate_client_onboarding_token_button"],
+            enable_screenshot=True,
+        )
+        strings_object_service_tab = [
+            "Client onboarding token",
+            "How to use this token",
+        ]
+        self.verify_page_contain_strings(
+            strings_on_page=strings_object_service_tab,
+            page_name="client_onboarding_token_page",
+        )
+
+    def verify_onboarding_token_generation_from_ui(self):
```

Comment:
this method is not verifying but 'generating' the token.
Please consider to change the name of the method to `generate_onboarding_token_generation_from_ui`

Reply:
removed